ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3RemoteServiceAdd.java

```diff
@@ -123,7 +123,7 @@ void installRuntimeServices(final OperationContext context, final ModelNode mode
                 .addDependency(AssociationService.SERVICE_NAME, AssociationService.class, ejbRemoteConnectorService.getAssociationServiceInjector())
                 .addDependency(TxnServices.JBOSS_TXN_REMOTE_TRANSACTION_SERVICE, RemotingTransactionService.class, ejbRemoteConnectorService.getRemotingTransactionServiceInjector())
                 .addDependency(ControlledProcessStateService.SERVICE_NAME, ControlledProcessStateService.class, ejbRemoteConnectorService.getControlledProcessStateServiceInjector())
-                .setInitialMode(ServiceController.Mode.ACTIVE);
+                .setInitialMode(ServiceController.Mode.ON_DEMAND);
```

Comment:
Because there is no reason for it to be started ACTIVEly.  This service is only needed by an EJB deployment with a remote view, and thus should only start when needed by that deployment.
Additionally, this service will now force the Registry to start, and we don't need to start any clustering services unless they are needed by a deployment.